tmp/root/cli_reference.ditamap

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please git ignore `tmp/`

Reply:
also added a user story to fix our template repos' gitignore